graphite_api/app.py

```diff
@@ -487,21 +516,24 @@ def evaluateTarget(requestContext, target):
     return result
 
 
-def evaluateTokens(requestContext, tokens):
+def evaluateTokens(requestContext, tokens, data_store):
     if tokens.expression:
-        return evaluateTokens(requestContext, tokens.expression)
+        return evaluateTokens(requestContext, tokens.expression, data_store)
 
     elif tokens.pathExpression:
-        return fetchData(requestContext, tokens.pathExpression)
+        return data_store.get_series_list(tokens.pathExpression)
 
     elif tokens.call:
         func = app.functions[tokens.call.funcname]
         args = [evaluateTokens(requestContext,
-                               arg) for arg in tokens.call.args]
+                               arg, data_store) for arg in tokens.call.args]
         kwargs = dict([(kwarg.argname,
-                        evaluateTokens(requestContext, kwarg.args[0]))
+                        evaluateTokens(requestContext,
+                                       kwarg.args[0],
+                                       data_store))
                        for kwarg in tokens.call.kwargs])
-        return func(requestContext, *args, **kwargs)
+        ret = func(requestContext, *args, **kwargs)
+        return ret
```

Comment:
No reason. I was probably just checking out the return value for some debugging purpose and forgot to change it back.